test/integration/feature/pester/minimal.package.Tests.ps1

```diff
@@ -18,5 +18,9 @@ describe 'test::feature' {
     it "feature Web-Asp-Net45 and Web-Net-Ext45 installed using powershell" {
       (Get-WindowsFeature Web-Asp-Net45,Web-Net-Ext45 | ?{$_.InstallState -eq "Installed"}).count | Should Be 2
     }
+
+    it "feature NPAS and management tools installed using powershell" {
+      (Get-WindowsFeature NPAS,NPAS-Policy-Server,RSAT-NPAS | ?{$_.InstallState -eq "Installed"}).count | Should Be 2
```

Comment:
I think you'll need `should be 3` for this test.

Reply:
Thanks, I'll update this. 

I was waiting for Appveyor on the tests since I don't have a super nice way to test locally, but looks like it's getting hung up on something else.